lib/ansible/modules/system/user.py

```diff
@@ -1001,9 +1001,13 @@ def create_homedir(self, path):
                     shutil.copytree(skeleton, path, symlinks=True)
                 except OSError as e:
                     self.module.exit_json(failed=True, msg="%s" % to_native(e))
-        else:
+            else:
+                try:
+                    os.makedirs(path)
+                except OSError as e:
+                    self.module.exit_json(failed=True, msg="%s" % to_native(e))
```

Comment:
This was an indentation bug, because it doesn't make sense to create the home dir, if it already exists, and it also doesn't make sense not to create it if `/etc/skell` doesn't exists, which was the case before.